L1Trigger/L1TMuonEndCap/src/bdt/Utilities.cc

```diff
@@ -49,7 +49,7 @@ const std::vector<double> emtf::twoJetsScale = std::vector<double>(twoJets_scale
 
 float processPrediction(float BDTPt, int Quality, float PrelimFit)
```

Comment:
Leave file as is.

Reply:
Since there's no changes except spaces, it should be trivial to review. I'd really prefer to have no trailing spaces and tabs.